bot/exts/moderation/infraction/management.py

```diff
@@ -45,25 +44,17 @@ def infractions_cog(self) -> Infractions:
     # region: Edit infraction commands
 
     @commands.group(name='infraction', aliases=('infr', 'infractions', 'inf', 'i'), invoke_without_command=True)
-    async def infraction_group(self, ctx: Context, infr_id: int = None) -> None:
-        """Infraction manipulation commands. If `infr_id` is passed then this command fetches that infraction."""
-        if infr_id is None:
+    async def infraction_group(self, ctx: Context, infraction: Infraction = None) -> None:
```

Comment:
We actually don't want this because then the error raised by the `Infraction` converter would get supressed (which means the `cog_command_error` handling wouldn't work).